tensorboard/plugins/hparams/api_test.py

```diff
@@ -288,14 +303,22 @@ def test_eager(self):
     start_pb = metadata.parse_session_start_info_plugin_data(start_plugin_data)
     end_pb = metadata.parse_session_end_info_plugin_data(end_plugin_data)
 
-    # Remove any dependence on system time.
-    start_pb.start_time_secs = 123.45
-    end_pb.end_time_secs = 234.56
+    # We're not the only callers of `time.time`; Keras calls it
+    # internally an unspecified number of times, so we're not guaranteed
+    # to know the exact values. Instead, we perform relative checks...
+    self.assertGreater(start_pb.start_time_secs, initial_time)
+    self.assertLess(start_pb.start_time_secs, end_pb.end_time_secs)
+    self.assertLess(start_pb.start_time_secs, final_time)
+    self.assertEqual(start_pb.start_time_secs % 1, initial_time % 1)
```

Comment:
Optional, but I'm not sure these last two asserts add much value, since all they really check (IMO) is that our mocked time advances 1 second at a time, which is a detail of the test, not the system under test.

Reply:
They’re intended to check that the fractional part doesn’t get dropped
in the pipeline. I can remove them if you prefer.

Reply:
I suppose.  Could we just test that instead by changing the mock to `+= 0.001` instead?